wallet_wrappers/Makefile

```
@@ -0,0 +1,169 @@
+android: android-arm64
+		@echo "All Android libraries built."
+
+all: ios android
+	cd ${WRAPPER_PATH}; dart run ffigen --config ffigen.yaml
+
+
+clean:
+		rm -rf ${OUT_DIR}
+
+
+.PHONY:
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)